### ISSUE TITLE
Discrepancy between API KDA and in game emblem KDA

### ISSUE THREAD

**Issue by paname75** [Bungie-net/api#384]
HI,
Since mayhem launched during the Dawning there is a discrepancy between the statistics returned by the K/AD emblem and the K/AD API response.

We saw several backend update and hotfix since then but this issue is still ongoing.
Do you know when this will be corrected/reflected in the API ? last API update was early December.

thanks

**Comment by vtubelsky** [comment]
probably related to the fact that mayhem games are not returned by the API at all:
https://github.com/Bungie-net/api/issues/348

**Comment by vthornheart-bng** [comment]
Yes, this is very likely unfortunately, and is a bummer... but in our next deployment, which we're targeting for early February, will have fixes for Mayhem and a number of other requested fixes in here.  Let's keep this open just in case there are edge cases we're not seeing where this has proved insufficient.

**Comment by paname75** [comment]
old but I found out that the emblems only count games you actually finished and didn't go to orbit before the end of the timer.
